masschallenge/impact-api
# AC-4706- Create schema endpoint for impact api

This branch adds the ability to get a schema for a given api request by sending an OPTIONS request to a particular endpoint. It also resolves some issues when the Organization models went in.

To test:

1 - checkout this branch and make sure you have the latest development checkout out for accelerate.

2 - load the development database by running `make dbload GZ_FILE=../accelerate/test_data/initial_schema.sql.gz`

3 - grant demoadmin access to the v1 apis via `make grant-permissions PERMISSION_USER=demoadmin@masschallenge.org PERMISSION_CLASSES=v1_clients`

4 - login as the demoadmin user and browse to a v1 endpoint:
     - /api/v1/organization/
     - /api/v1/user/

5 - the interface should now allow you to make an 'OPTIONS' call to the endpoint which should return a schema with field definitions for each object.

![screen shot 2017-08-23 at 11 51 17 am](https://user-images.githubusercontent.com/196425/29625370-7babc17a-87f9-11e7-854e-98ef849776bf.png)
